src/argilla/server/schemas/v1/questions.py

```diff
@@ -86,3 +90,21 @@ class Question(BaseModel):
 
     class Config:
         orm_mode = True
+
+
+class TextQuestionSettingsUpdate(UpdateSchema):
+    use_markdown: Optional[bool]
+
+    __non_explicit_none__ = {"use_markdown"}
+
+
+class LabelSelectionSettingsUpdate(UpdateSchema):
+    visible_options: Optional[int]
+
+
+class QuestionUpdate(UpdateSchema):
+    title: Optional[str]
+    description: Optional[str]
+    settings: Optional[Union[Dict[str, Any], TextQuestionSettingsUpdate, LabelSelectionSettingsUpdate]]
```

Comment:
why not use the same approach as the `QuestionCreate` schema? 
```python
QuestionSettingsUpdate = Annotated[
    Union[
        TextQuestionSettingsUpdate,
        LabelSelectionQuestionSettingsUpdate,
        MultiLabelSelectionQuestionSettingsUpdate,
    ],
    PydanticField(discriminator="type"),
]

```

Reply:
I didn't do it this way, so the client didn't need to provide the `settings.type` of questions which will be updated, but it makes sense for the client to provide the `settings.type` as it will know the type of question to be updated.